packages/gatsby/src/utils/node-manifest.ts

```diff
@@ -259,21 +235,20 @@ export async function processNodeManifest(
     nodeId,
   })
 
-  warnAboutNodeManifestMappingProblems({
-    inputManifest,
-    pagePath: nodeManifestPage.path,
-    foundPageBy,
-  })
+  if (verboseLogs || !warnOnceAboutNodeManifestMappingProblems) {
+    warnAboutNodeManifestMappingProblems({
```

Comment:
I think instead of warning once we should collect the types of warnings and after processing everything we can log out a message combining them. In verbose mode we can list each type and in quiet mode we can link to docs - but we should probably still warn in quiet mode if no page was found for a node.

Reply:
I'm trying to decide how to combine it and what the messaging here should be. Right now there are two logs to combine, one is a warning `Plugin ${inputManifest.pluginName} called unstable_createNodeManifest for a node which doesn't exist with an id of ${nodeId}.` and the other is a structured error
```
      logId = foundPageByToLogIds[foundPageBy]
      reporter.error({
        id: logId,
        context: {
          inputManifest,
          pagePath,
        },
      })
```
Or are you saying to remove the structured error and just list the types errors (in verbose) or just a link to the docs in quiet mode.

Reply:
In verbose mode we want to print out every warning/error but in quiet mode I think we should print a single message at the end something like: `unstable_createNodeManifest produced warnings [${commaSeparatedListOfUniqueErrorIds}]. Visit https://gatsby.dev/nodemanifest for more info.`.

We could then have a page that lists out what each error ID means and how to enable verbose mode (`Set process.env.VERBOSE_CREATE_NODE_MANIFEST to "true" to see each warning/error message`)

Reply:
The nice thing about having a page for this is we can explain what the warnings mean in greater detail. A lot of people think these warnings are bugs in createNodeManifest rather than a message explaining to them how they can make it more accurate.